docs/excel/custom-functions-dialog.md

```diff
@@ -1,22 +1,25 @@
 ---
-ms.date: 03/21/2019
+ms.date: 034/28/2019
 description: Create dialog boxes through custom functions in Excel using JavaScript.
 title: Custom functions dialogs (preview)
 localization_priority: Priority
 ---
 # Display a dialog box in custom functions
 
-If your custom function needs to interact with the user, you can create a dialog box using the `OfficeRuntime.Dialog` object. A common scenario for using the dialog box is to authenticate a user so that your custom function can access a web service. For more details about authentication with custom functions, see [Custom functions authentication](./custom-functions-authentication.md).
+If your custom function needs to interact with the user, you can create a dialog box using the `Office.Dialog` object. A common scenario for using the dialog box is to authenticate a user so that your custom function can access a web service. For more details about authentication with custom functions, see [Custom functions authentication](./custom-functions-authentication.md).
```

Comment:
Can we change the first reference to Office.Dialog to a link to the Dialog object reference page?

Reply:
Good idea!